raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -759,7 +774,12 @@ void assertSentBeginQuorumEpochResponse(
 
         assertEquals(epoch, partitionResponse.leaderEpoch());
         assertEquals(leaderId.orElse(-1), partitionResponse.leaderId());
-        assertEquals(partitionError, Errors.forCode(partitionResponse.errorCode()));
+        assertEquals(
+            partitionError,
+            Errors.forCode(partitionResponse.errorCode()),
+            "Lead Id: " + leaderId +
+                " Partition response leader Id: " + partitionResponse.leaderId()
```

Comment:
I think this is the correct indentation.
```java
            "Lead Id: " + leaderId +
            " Partition response leader Id: " + partitionResponse.leaderId()
```

Reply:
Thanks for the feedback @jsancio . Updated in latest commit.